samples/productSearch/system-test/.eslintrc.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@beccca @kinwa91 I think this is the first place we're putting samples two levels deep like this.  Is this going to work with our kokoro setup?

Reply:
Adding the right id:  @beccataylor